native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/tasks/BuildNativeImageTask.java

```diff
@@ -190,7 +190,8 @@ private List<String> buildActualCommandLineArgs(int majorJDKVersion) {
                 getProviders().provider(() -> getOutputDirectory().getAsFile().get().getAbsolutePath()),
                 getClasspathJar(),
                 getUseArgFile(),
-                getProviders().provider(() -> majorJDKVersion)).asArguments();
+                getProviders().provider(() -> majorJDKVersion),
+                getProviders().provider(() -> "plain".equals(getProject().getGradle().getStartParameter().getConsoleOutput()))).asArguments();
```

Comment:
:x: Calling `getProject()` at execution time is not compatible with Gradle's configuration cache. This must be changed, and the easiest is to copy the value of the `getConsoleOutput()` directly in the task constructor, so that you don't have to query that at execution time.

Reply:
Thanks for catching! I've fixed this, please take another look.